### DISCUSSION TITLE
How to specify negative prompt when using gen_img_diffusers.py?

### DISCUSSION THREAD

**Discussion by wcc19840827** [kohya-ss/sd-scripts#218] [Q&A]
How to specify negative prompt when using gen_img_diffusers.py?

**Comment by ReppiksProductions** [comment]
In case you or anyone else are still wondering how to do this.  --n: Specifies a negative prompt.

Take a look at this doc here that really breaks down everything you can do with gen_img_diffusers.py.  https://github.com/darkstorm2150/sd-scripts/blob/main/docs/gen_img_README-en.md

It's a translation from Kohya's original ReadMe
